### DISCUSSION TITLE
Список друзей

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#21] [Ideas]
Предлагаю добавить в API класс для списка с друзьями, например FriendsList. Например там будет метод Add(player). И при вызове метода если игрок есть на сервере, то ему выскажет окошко с подтверждением.

**Comment by kkohno** [comment]
это может привести к томы, что у Вас в друзьях появится вся комната, без Вашего участия... А концепция в том, что в друзья Вы сами выбираете кого добавить вручную а не кто-то за Вас
